docs/topics/forms/index.txt

```diff
@@ -733,12 +733,17 @@ Reusable form templates
 
 If your site uses the same rendering logic for forms in multiple places, you
 can reduce duplication by saving the form's loop in a standalone template and
-using the :ttag:`include` tag to reuse it in other templates:
+using over ride your forms ``__str__`` method to reuse it in other templates:
```

Comment:
This needs rephrasing entirely. We should encourage that `template_name` is overridden as is the easiest way to achieve this...